src/lib.rs

```diff
@@ -350,9 +350,9 @@ pub fn parse_header(raw_data: &[u8]) -> Result<(MailHeader, usize), MailParseErr
     match ix_key_end {
         Some(v) => {
             Ok((MailHeader {
-                key: &raw_data[0..v],
-                value: &raw_data[ix_value_start..ix_value_end],
-            },
+                    key: &raw_data[0..v],
```

Comment:
I try to keep formatting changes separate from functional changes, because otherwise it makes changes harder to review and harder to understand when looking back in history. And with rust code in particular, I always do formatting by just running the rust formatter and committing the result as a single changeset with commit message "rustfmt". Please do the same (assuming this formatting change is generated by rustfmt; if not then please don't make it).